tests/foreman/installer/test_installer.py

```diff
@@ -1359,6 +1359,7 @@ def test_positive_check_installer_services(target_sat):
         assert result['Status'] == 'ok', f'{service} responded with {result}'
 
 
+@pytest.mark.e2e
```

Comment:
For this one we will need to make sure that we are tracking the failures for this on stream and updating master with new and removed options.

Reply:
yeah, we'll have to fix this one.